tests/Proto.Actor.Tests/FutureTests.cs

```diff
@@ -102,5 +101,41 @@ public void Given_Actor_When_ReplyIsNull_Should_Return()
                 output.WriteLine("got response 2");
             }).Wait();
         }
+
+        [Fact]
+        public void TestInATaskIndirect() // This one will break
```

Comment:
Shall we remove this comment and get it merged?

Reply:
removed